tool_aws/s3/rm.py

```diff
@@ -155,7 +155,8 @@ def createParser():
         action='store',
         type=bboxType,
         default=None,
-        help='a bounding box in lv95')
+        help='a bounding box in lv95. Only works in combination with  \
```

Comment:
is it this way round or the other?

Reply:
Both are needed, if with bbox. 

And if only prefix it is misleading to put -i (as it has no affect)

Reply:
no, I don't think so